addons/packages/antrea/0.11.3/bundle/config/overlay/antrea_overlay.yaml

```diff
@@ -169,44 +169,15 @@ metadata:
     kapp.k14s.io/disable-default-label-scoping-rules: ""
 
 
-#@overlay/match by=overlay.subset({"kind":"DaemonSet","metadata":{"name": "antrea-agent"}})
+#@overlay/match by=overlay.subset({"kind": "DaemonSet", "metadata": {"name": "antrea-agent"}})
 ---
-kind: DaemonSet
```

Comment:
@hangyan Is it expected to remove `kind: DaemonSet`? Can this patch be located correctly? I see other patches have kind.

Reply:
Yes, they both works. I have tested it.

Reply:
Could we keep the kind like the above Deployment kind? After removing the update to "spec", they look similar and should keep align.